src/components/custom/dashboard/trainer/pages/newclients/components/RequestCard.jsx

```
@@ -0,0 +1,254 @@
+      )}
+
+      {/* Main Card Content */}
+      <div className="flex items-center p-4 gap-4">
+        {/* Profile Section */}
+        <div className="flex items-center gap-3 min-w-0 flex-1">
+          {/* Profile Image */}
+          <button
+            className="relative cursor-pointer group/image flex-shrink-0"
+            tabIndex="0"
+            type="button"
+            role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)